apps/davi/src/stores/modules/SnapshotRepGuild/index.ts

```diff
@@ -53,7 +54,7 @@ export const snapshotRepGuildImplementation: Readonly<FullGovernanceImplementati
           useProposalCalls,
           useVotingResults,
           useVotingPowerOf,
-          useGuildConfig,
+          useGuildConfig: useGuildConfigFromSubgraph,

```

Comment:
This should be duplicated in non rep guilds also

Reply:
Also when added manually in there I seem to get 404s in the network tab

Reply:
> Also when added manually in there I seem to get 404s in the network tab

Where does this happen and how to reproduce this?

Reply:
Fixed with the uri update